subsys/bluetooth/Kconfig

```diff
@@ -83,6 +83,34 @@ config BT_BROADCASTER
 
 endmenu
 
+config BT_EXT_ADV
+	bool "Extended Advertising and Scanning support [EXPERIMENTAL]"
+	help
+	  Select this to enable Extended Advertising API support.
+	  This enables support for advertising with multiple advertising sets,
+	  extended advertising data, and advertising in long range mode.
+	  It enables support for receiving and scanning extended advertising
+	  data, and packets sent in long range mode.
+	  It enables establishing connections in long range mode.
+
+if BT_EXT_ADV
+config BT_EXT_ADV_LEGACY_SUPPORT
+	bool "Support starting advertising through legacy commands"
+	help
+	  Select this to enable the use of the Legacy Advertising HCI commands.
+	  This option should be used where the capabilities of the controller
+	  is not known.
```

Comment:
Add: If this option is not enabled the controller must support the extended advertising feature.